cmd/agent/app/integrations.go

```diff
@@ -216,6 +225,24 @@ func tuf(args []string) error {
 		tufCmd.Env = append(tufCmd.Env,
 			fmt.Sprintf("TUF_CONFIG_FILE=%s", tufPath),
 		)
+
+		// Enable tuf logging
+		if tufLog {
+			tufCmd.Env = append(tufCmd.Env,
+				"TUF_ENABLE_LOGGING=1",
```

Comment:
long-term, is there any reason for TUF to not enable its logging when pip is run with verbose logging?

Reply:
That's a good suggestion: I'll make a note to take `pip -vvv` as a cue to turn on TUF logging.